### PR TITLE
Add section id as url param to back to level link

### PR DESCRIPTION
**Pull request by dju90**
...on CFU summary page.

## Links

- jira ticket: [TEACH-387](https://codedotorg.atlassian.net/browse/TEACH-387)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/levelSummary/CheckForUnderstanding.jsx [hunk 1]
```diff
@@ -23,6 +23,9 @@ const CheckForUnderstanding = ({
 }) => {
   const currentLevel = levels.find(l => l.activeId === currentLevelId);
   const nextLevel = levels.find(l => l.position === currentLevel.position + 1);
+  const sectionParam = selectedSection?.id
```
Comment (rshipp) [inline apps/src/templates/levelSummary/CheckForUnderstanding.jsx:26]: whoa, i didn't realize we had access to this `?.` operator yet!

File: apps/src/templates/levelSummary/CheckForUnderstanding.jsx [hunk 2]
```diff
@@ -64,7 +67,7 @@ const CheckForUnderstanding = ({
     <div className={styles.summaryContainer}>
       {/* Top Nav Links */}
       <p className={styles.navLinks}>
-        <a href={currentLevel.url} onClick={onBackToLevelClick}>
+        <a href={`${currentLevel.url}${sectionParam}`} onClick={onBackToLevelClick}>
```
Comment (rshipp) [inline apps/src/templates/levelSummary/CheckForUnderstanding.jsx]: can you update this url in `onBackToLevelClick` above, too?

**Comment by rshipp** [comment]
did you test this with other params in the URL, like `?viewAs=Instructor`? i think it'd be fine, but just to make sure

also, do we need this to apply to the "Next" link as well?

File: apps/src/templates/levelSummary/CheckForUnderstanding.jsx (hunk 1 above)
Comment (dju90) [reply apps/src/templates/levelSummary/CheckForUnderstanding.jsx:26]: I think it's just in the apps directory (https://codedotorg.slack.com/archives/C0T0PNR0D/p1680630902632839?thread_ts=1680562949.098389&cid=C0T0PNR0D)

**Comment by dju90** [comment]
Urgh, this PR contains so many changed files due to a merge. Closing for now and creating a fresh PR 🤦

**Pull request closed without merging**